lib/msal-common/src/telemetry/performance/PerformanceEvent.ts

```diff
@@ -221,6 +221,12 @@ export type StaticFields = {
     matsHttpEventCount?: number;
     httpVerToken?: string;
     httpVerAuthority?: string;
+
+    // Broker timeouts used by 1p browser lib
+    brokerInteractionTimeout?: number;
```

Comment:
I'm assuming these are all in ms?  Suggest renaming them to brokerInteractionTimeoutMs, etc.

Reply:
Good point, thank you. Done.